File: safeeye+/src/fiche/safeeye/entities/Fiche.java

package fiche.safeeye.entities;

/**
 *
 * @author 21654
 */
public class Fiche {
        private int id_fiche;
        private String nom_patient;
        private String prenom_patient;
        private String medecin;
        private String maladie;
    
    public Fiche (int id_fiche ,String nom_patient ,String prenom_patient ,String medecin ,String maladie){
        this.id_fiche=id_fiche;
        this.nom_patient=nom_patient;
        this.prenom_patient=prenom_patient;
        this.medecin=medecin;
        this.maladie=maladie;
    }
    public Fiche ( String nom_patient, String prenom_patient ,String medecin, String maladie){
        this.nom_patient=nom_patient;
        this.prenom_patient=prenom_patient;
        this.medecin=medecin;
        this.maladie=maladie;
     
    }

    public int getId_fiche() {
        return id_fiche;
    }

    public String getNom_patient() {
        return nom_patient;
    }
      public String getPrenom_patient() {
        return prenom_patient;
    }
      public String getMedecin() {
        return medecin;
    }
        public String getMaladie() {
        return maladie;
    }
    public void setId_fiche(int id_fiche) {
        this.id_fiche = id_fiche;
    }
    public void setNom_patient(String nom_patient) {
        this.nom_patient=nom_patient;
    }
    public void setPrenom_patient(String prenom_patient) {
        this.prenom_patient=prenom_patient;
    }

    public void setMedecin(String medecin) {
        this.medecin = medecin;
    }
   public void setMaladie(String maladie) {
        this.maladie=maladie;
    }

    @Override
    public String toString() {
        return "Fiche{" + "id fiche=" + id_fiche + ", Nom_patient=" + nom_patient +
                ", Prenom_patient=" + prenom_patient +  "Medecin=" + medecin + "Maladie=" +maladie +'}';
    }

    public Fiche() {
    }


}

File: src/Entity/Fiche.php
<?php

namespace App\Entity;

use App\Repository\FicheRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: FicheRepository::class)]
class Fiche
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(length: 255)]
    private ?string $Nom_patient = null;

    #[ORM\Column(length: 255)]
    private ?string $Prenom_patient = null;

    #[ORM\Column(length: 255)]
    private ?string $Medecin = null;

    #[ORM\Column(length: 255)]
    private ?string $Maladie = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomPatient(): ?string
    {
        return $this->Nom_patient;
    }

    public function setNomPatient(string $Nom_patient): self
    {
        $this->Nom_patient = $Nom_patient;

        return $this;
    }

    public function getPrenomPatient(): ?string
    {
        return $this->Prenom_patient;
    }

    public function setPrenomPatient(string $Prenom_patient): self
    {
        $this->Prenom_patient = $Prenom_patient;

        return $this;
    }

    public function getMedecin(): ?string
    {
        return $this->Medecin;
    }

    public function setMedecin(string $Medecin): self
    {
        $this->Medecin = $Medecin;

        return $this;
    }

    public function getMaladie(): ?string
    {
        return $this->Maladie;
    }

    public function setMaladie(string $Maladie): self
    {
        $this->Maladie = $Maladie;

        return $this;
    }
}
